src/libs/ui/components/list/list.tsx

```diff
@@ -77,12 +79,14 @@ export function List<ListRow extends ListRowBase>({
   renderFooter,
   headerLabel,
   headerAction,
-  marginLeftForItemSeparatorLine
+  marginLeftForItemSeparatorLine,
+  headerLabelSpaceSize = SpacingSize.ExtraLarge,
+  contentSpaceSize = SpacingSize.ExtraLarge
```

Comment:
this change seems unnecessary as all props are changed to use the same value Small, so it seems to me like a simple refactor instead of a component design change.

Reply:
Not all. I only passed these props, `contentSpaceSize` and `headerLabelSpaceSize`, a few times where it's needed by design. These default values are for all other cases when we didn't pass these props.

Reply:
In that case, seems like we have most 3 variations, in my opinion adding variations makes more sense than adding unnecessary complexity to the component API